docs/source/docs/explanation/formats/segment_anything.md

```diff
@@ -0,0 +1,117 @@
+# Segment Anything
+
+## Format specification
+
+SA-1B format specification is available [here](https://github.com/facebookresearch/segment-anything#dataset)
+
+The SA-1B is an dataset consisting of 1 billion segmentation masks model generated.
```

Comment:
just recommendation.
model generated
->
from interactions between human annotators and the trained model.

Reply:
The annotations in SA-1B are from SAM generated ones only. There is no human-in-the-loop in this particular dataset though 2 stages out of 3 stages SAM was trained include the human-in-the-loop.

From the paper:
> this dataset includes only automatically generated masks from the final stage of our data engine.